src/main/java/core/basesyntax/ReportGenerationImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReportGenerationImpl implements ReportGenerationMet {
+    private final String headerForResult = "fruit,quantity";
+    private final String comma = ",";
+
+    @Override
+    public String reportGeneration(Map<String, Integer> storage) {

```

Comment:
Storage represents real DB. So create a separate Storage class with a public static map and call it directly inside the method.